### PR TITLE
adding explicit require for bluebird Promise; removing deprecated class

### PR DESCRIPTION
**Pull request by UnsungHero97**

### CODE REVIEW

File: js/src/workers/WorkerWrapper.coffee [hunk 1]
```diff
@@ -1,8 +1,12 @@
 define 'kryptnostic.worker-wrapper', [
   'require',
+  'bluebird',
   'kryptnostic.logger'
 ], (require) ->
 
+  # libraries
+  Promise = require 'bluebird'
```
Comment (synecdoche-nh) [inline js/src/workers/WorkerWrapper.coffee]: LGTM

Can you create channels in IE11 with this fix? 
Or does the same error log appear as attached in PLATFORM-111?

Comment (UnsungHero97) [inline js/src/workers/WorkerWrapper.coffee]: creating channels seems to work for me, but I still see the same errors being logged

**Comment by synecdoche-nh** [comment]
that's good news, but curious that same errors exist

**Comment by UnsungHero97** [comment]
I have a fix for it in another PR

**Pull request merged**